libr/bin/p/bin_elf.inc

```diff
@@ -1118,7 +1102,7 @@ static RBinInfo* info(RBinFile *bf) {
 	if (!(ret = R_NEW0 (RBinInfo))) {
 		return NULL;
 	}
-	ret->lang = is_cpp (bf) ? "c++" : "c";
+	ret->lang = r_bin_is_cxx (bf) ? "cxx" : "c";
```

Comment:
I think this change actually changes the semantic of the check... before we were just checking imports, now you are checking all symbols, right? Is that what you wanted?

Also, shall we consider C binaries compiled with ASAN (and that for this reasons have some mangled names) C++ binaries? Or shall we filter those ASAN symbols out?

Reply:
No after second thoughts I prefer not considering C sanitized binaries be seen as.